ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProvidersList.tsx

```diff
@@ -67,51 +67,61 @@ function AuthProvidersList({ entityId, authProviders }: AuthProvidersListProps):
                     </Tr>
                 </Thead>
                 <Tbody>
-                    {authProviders.map(({ id, name, type, defaultRole, groups = [] }) => {
-                        const typeLabel = getAuthProviderTypeLabel(type, availableProviderTypes);
+                    {authProviders.map(
+                        ({ id, name, type, defaultRole, unmodifiable, groups = [] }) => {
```

Comment:
An observation, not necessarily a change request: here is an idiom to avoid the change in indentation which makes it hard to find the few changes below.

```js
{authProviders.map((authProvider) => {
    const { id, name, type, defaultRole, unmodifiable, groups = [] } = authProvider;
```